spec/features/users_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Users', type: :feature do
+  context 'registrations users' do
+    scenario 'empty email and password' do
+      visit new_user_registration_path
+      click_button 'Sign up'
+
+      expect(page).to have_content("Email can't be blank")
+      expect(page).to have_content("Password can't be blank")
+    end
+
+    scenario 'wrong email' do
+      visit new_user_registration_path
```

Comment:
Seema that `visit new_user_registration_path`, `fill_in ...` and `click_button ...` actions are duplicated. Could you move them to `before` block?